userProfile/src/Components/RenderModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, Modal } from '@skbkontur/react-ui';
+import React from 'react';
+import { FormData } from './Form';
+
+type TProps = {
+    savedData: FormData;
+    person: FormData;
+    onClose: Function;
+};
+
+const RenderModal = (props: TProps) => {

```

Comment:
Деструктуризацию можно делать прямо в объявлении пропсов
```suggestion
const RenderModal = ({ savedData, person, onClose }: TProps) => {
```